pkg/projget/projget.go

```diff
@@ -0,0 +1,58 @@
+package projget
```

Comment:
Why is this in its own package?

Reply:
A story has been filed to remove this package: https://activestatef.atlassian.net/browse/DX-1808